addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -6,22 +6,74 @@
              data-vcss="001"
              data-columns="3" style="height: 500px; overflow: hidden;">
         <div class="container">
-            <div class="alert alert-info css_non_editable_mode_hidden text-center" role="status"><span class="o_add_images" style="cursor: pointer;"><i class="fa fa-plus-circle"/> Add Images</span></div>
+          <div id="slideshow_sample" class="carousel slide" data-ride="carousel" data-interval="0" style="margin: 0 12px;">
+            <div class="carousel-inner" style="padding: 0;">
+              <div class="carousel-item active" style="">
+                <img class="img img-fluid d-block" src="/web/image/website.library_image_08" style="" alt="" data-name="Image" data-index="0"/>
+              </div>
+              <div class="carousel-item undefined" style="">
```

Comment:
`carousel-item undefined` ? `style=""` ? Also the indentation is to be reviewed.